javascript/javascript2/week1/exercises.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="EN">
+    <head>
+        <meta charset="UTF-8">
+        <link rel="stylesheet" href="style.css">
+        
+    </head>
+    <body>
+        <h1> js 2 w 1</h1>
+
+        <div class="events"></div>
+        <div id="logo"></div>
+        <button class="btn-class1">click me</button>
+        <button class="btn-class2">change color</button>
```

Comment:
These class-names should be more descriptive. Eg. eventlistener-button and color-change-button

Reply:
yes it was for practice and test